### PR TITLE
Add band to concepts, address issue 248 items on catalog doc page

### PR DESCRIPTION
**Pull request by vpipkt**
@courtney-whalen can you take a look. Referencing #248 items,

* haven't explicitly addressed the experimental descriptor on the catalogs
* i did a show (head) on the earlier external catalog example. We'd rather not do it on the built in ones because they can be slow to initialize

### CODE REVIEW

**Comment by courtney-whalen** [comment]
I went through the changes - looks good to me!

**Comment by courtney-whalen** [comment]
I can't close #248 though. I think I only have permission to close an issue if I created the issue.

File: pyrasterframes/src/main/python/docs/raster-catalogs.pymd [hunk 1]
```diff
@@ -25,30 +25,37 @@ A single URL is the simplest form of a catalog.
...
+
+file_uri = "/data/raster/myfile.tif"
+# Pandas DF
+my_cat = pd.DataFrame({'B01': [file_uri]})
+
+# equivalent Spark DF
+my_cat = spark.createDataFrame([Row(B01=file_uri)])
+
+#equivalent CSV string
+my_cat = "B01\n{}".format(file_uri)
```
Comment (metasim) [inline pyrasterframes/src/main/python/docs/raster-catalogs.pymd]: :win:

File: pyrasterframes/src/main/python/docs/raster-catalogs.pymd [hunk 2]
```diff
@@ -25,30 +25,37 @@ A single URL is the simplest form of a catalog.
...
-my_cat = spark.createDataFrame(pandas_cat)
+# a pandas DF
+one_d_cat = pd.DataFrame({'B01': [scene1_B01, scene2_B01]})
+
+# equivalent spark DF
+one_d_cat = spark.createDataFrame([Row(B01=scene1_B01), Row(B01=scene2_B01)])
+
+# equivalent CSV string
+one_d_cat = '\n'.join(['B01', scene1_B01, scene2_B01])
 ```
```
Comment (metasim) [inline pyrasterframes/src/main/python/docs/raster-catalogs.pymd]: ❤️

**Comment by vpipkt** [comment]
:shipit:

**Pull request merged**